pkg/rtcp/slice_loss_indication.go

```diff
@@ -0,0 +1,115 @@
+package rtcp
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+)
+
+// The SliceLossIndication packet informs the encoder about the loss of a picture slice
+type SliceLossIndication struct {
+	// SSRC of sender
+	SenderSSRC uint32
+
+	// SSRC of the media source
+	MediaSSRC uint32
+
+	SLI []struct {
```

Comment:
It might be helpful to make this an exported struct type instead of an anonymous one.

It's difficult to, for instance, maintain a list of SLIs if there's no type name.